articles/libraries/secure-local-development.md

```diff
@@ -0,0 +1,68 @@
+---
+section: libraries
+description: Securing local development servers to work with samesite cookies
+topics:
+  - libraries
+  - samesite
+contentType:
+  - reference
+---
+
+# Secure Local Development
+Google is introducing new restrictions on cookie usage in version 80 of Chrome with a new cookie attribute named [Same Site](https://auth0.com/blog/browser-behavior-changes-what-developers-need-to-know/).  This article covers how it impacts using Auth0 in your local environment with certain SDKs.
+
+This article only impacts you if you are using one of the following SDKS:
+- express-openid-connect
+- another one
+
+These SDKs utilize cookies to handle request state and prevent replay attacks.  When the authentication request is handed back from Auth0 to your local server, the new same site restrictions will prevent the cookie being passed if the local server is not secure (https).  Without this cookie value, the local server will be unable to complete the authentication request.
+
+## How to set up a secure local environment
+Instructions on how to set up a secure local environment can be daunting and burdensome.  However, thanks to the open source community, [Mkcert](https://github.com/FiloSottile/mkcert) is a tool that automates and simplifies the implementation.
+
+Follow the [installation instructions](https://github.com/FiloSottile/mkcert#installation) for your specific operating system.
+
+### Installing the local Certificate Authority
+After you have mkcert installed you can use it to install the local Certificate Authority.  The local Certificate Authority will allow for mkcert to generate locally-trusted certificates and accompanying keys to allow your local development server to run on `https`.
+
+```powershell
+> mkcert -install
+# Using the local CA at "/Users/$HOME/Library/Application Support/mkcert" ✨
+# The local CA is now installed in the system trust store! 👍
+# The local CA is now installed in the Firefox trust store (requires browser restart)! 🦊
+```
+
+### Generate a locally-trusted certificate & key
+The next step is to generate the locally-trusted certificate and key for use on your local development server. This example will assume you are running your local server on `https://localhost:{port}` or `https://127.0.0.1:{port}`.  Multiple values can be passed into mkcert to support a range of address names.
```

Comment:
I wonder if if would be worth it to add a note/panel to explain a bit more about what's going on here with the domains. Maybe also make note of the fact that the localhost one will cover quickstarts.